Problem at link of fortran tests with intel compilers

When building with the intel compilers by
```
export CC=icc
export FC=ifort
export CXX=icpc
```
before issuing
```
mkdir build
cd build
cmake -DQV_DISABLE_GPU_SUPPORT=ON -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON ..
make -j 6
```
the compilation fails when linking a test fortran main with
```
[ 90%] Linking CXX executable test-mpi-qvfort
cd /home/globc/andrea/WORK/QUO-VADIS/quo-vadis/build/tests && /home/globc/andrea/INSTALL/cmake/3.22.0/bin/cmake -E cmake_link_script CMakeFiles/test-mpi-qvfort.dir/link.txt --verbose=1
/scratch/root/opt/intel/oneapi/compiler/2021.1.1/linux/bin/intel64/icpc  -Wall -Wextra -pedantic -g CMakeFiles/test-mpi-qvfort.dir/test-mpi-qvfort.f90.o -o test-mpi-qvfort  ../src/fortran/libquo_vadis_mpif.a ../src/libquo-vadis-mpi.a ../src/libqvi-mpi.a ../src/fortran/libquo_vadisf.a ../src/libquo-vadis.a ../src/libqvi-rmi.a ../src/libqvi-hwloc.a ../src/libqvi-core.a ../spdlog/lib/libspdlog.a -pthread ../hwloc/lib/libhwloc.a /usr/lib64/libpciaccess.so /usr/lib64/libzmq.so /scratch/root/opt/intel/oneapi/mpi/2021.1.1/lib/libmpifort.so /scratch/root/opt/intel/oneapi/mpi/2021.1.1/lib/release/libmpi.so /usr/lib64/libdl.so /usr/lib64/librt.so -lpthread -Xlinker --enable-new-dtags -Xlinker -rpath -Xlinker '/scratch/root/opt/intel/oneapi/mpi/2021.1.1/lib/release' -Xlinker -rpath -Xlinker '/scratch/root/opt/intel/oneapi/mpi/2021.1.1/lib' -L/home/globc/andrea/WORK/QUO-VADIS/quo-vadis/'/scratch/root/opt/intel/oneapi/mpi/2021.1.1/lib/release' -L/home/globc/andrea/WORK/QUO-VADIS/quo-vadis/'/scratch/root/opt/intel/oneapi/mpi/2021.1.1/lib' -lifport -lifcoremt -lpthread 
/usr/lib/gcc/x86_64-redhat-linux/7/../../../../lib64/crt1.o: In function `_start':
(.text+0x20): undefined reference to `main'
make[2]: *** [tests/CMakeFiles/test-mpi-qvfort.dir/build.make:116: tests/test-mpi-qvfort] Error 1
make[2]: Leaving directory '/home/globc/andrea/WORK/QUO-VADIS/quo-vadis/build'
make[1]: *** [CMakeFiles/Makefile2:1437: tests/CMakeFiles/test-mpi-qvfort.dir/all] Error 2
make[1]: Leaving directory '/home/globc/andrea/WORK/QUO-VADIS/quo-vadis/build'
make: *** [Makefile:169: all] Error 2

```

Thanks
Andrea

--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -84,6 +84,12 @@ target_link_libraries(
     test-mpi-qvfort
     quo_vadis_mpif
 )
+
+set_target_properties(
+  test-mpi-qvfort
+  PROPERTIES LINKER_LANGUAGE Fortran
+)
+
 endif(QV_FORTRAN_HAPPY AND MPI_Fortran_FOUND)
 
 ################################################################################

